### PR TITLE
Disable chapter break by default

### PR DESCRIPTION
**Pull request by fcecagno**

### CODE REVIEW

**Comment by capilkey** [comment]
Should also change the default here, https://github.com/bigbluebutton/bigbluebutton/blob/cb0d28245133402275826812084eccbbb533dea7/akka-bbb-apps/src/main/scala/org/bigbluebutton/SystemConfiguration.scala#L43.

File: akka-bbb-apps/src/main/scala/org/bigbluebutton/SystemConfiguration.scala [hunk 1]
```diff
@@ -40,7 +40,7 @@ trait SystemConfiguration extends RedisConfiguration {
   lazy val applyPermissionCheck = Try(config.getBoolean("apps.checkPermissions")).getOrElse(false)
 
   lazy val voiceConfRecordPath = Try(config.getString("voiceConf.recordPath")).getOrElse("/var/freeswitch/meetings")
-  lazy val recordingChapterBreakLenghtInMinutes = Try(config.getInt("recording.chapterBreakLengthInMinutes")).getOrElse(180)
+  lazy val recordingChapterBreakLenghtInMinutes = Try(config.getInt("recording.chapterBreakLengthInMinutes")).getOrElse(0)
```
Comment (kepstin) [inline akka-bbb-apps/src/main/scala/org/bigbluebutton/SystemConfiguration.scala]: While you're in here, could you fix the typo in the variable name? It says "Lenght", but should be "Length".

**Comment by kepstin** [comment]
I'm not familiar with the code in question, but I don't see anything obviously wrong and I like the choice of "0" as a disabled value.

File: akka-bbb-apps/src/main/scala/org/bigbluebutton/SystemConfiguration.scala (hunk 1 above)
Comment (fcecagno) [reply akka-bbb-apps/src/main/scala/org/bigbluebutton/SystemConfiguration.scala]: Done.

**Pull request merged**